backend/src/main/java/com/znane_groby/backend/model/City.java

```diff
@@ -0,0 +1,39 @@
+package com.znane_groby.backend.model;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.*;
+import org.springframework.hateoas.RepresentationModel;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
```

Comment:
Chyba `@Data` to wszystko robi za jednym razem (najwyżej brakuje konstruktora jakiegoś). to string, hashcode i equals w gratisie.

Reply:
Norn, ale jesteś tego pewien? Bo znalazłem coś co mówi, że nie można mieszać @data i @entity (o ile nie zoveraidujesz hasha i equals, co zaczyna sie mijac z celel.
https://mdeinum.github.io/2019-02-13-Lombok-Data-Ojects-Arent-Entities/
Prawda to, czy jakiś bullshit?